src/mds/FSMap.cc

```diff
@@ -1090,10 +1090,13 @@ std::vector<mds_gid_t> FSMap::stop(mds_gid_t who)
     if (other_info.rank == info.rank
         && other_info.state == MDSMap::STATE_STANDBY_REPLAY) {
       standbys.push_back(other_gid);
-      erase(other_gid, 0);
     }
   }
 
+  for (const auto &other_gids : standbys) {
+    erase(other_gids, 0);
+  }
+
```

Comment:
nit: Aliaksei, could you please change `other_gids` to a singular noun i.e. `other_gid`

Reply:
Done!